plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol

```diff
@@ -127,59 +128,59 @@ contract PaymentInFlightExitRouter is
         uint168 indexed exitId
     );
 
-    constructor(PaymentExitGameArgs.Args memory args)
-        public
+    function boot(PaymentExitGameArgs.Args memory paymentExitGameArgs) 
+        internal
     {
-        framework = args.framework;
-
-        EthVault ethVault = EthVault(args.framework.vaults(args.ethVaultId));
+        require(msg.sender == paymentExitGameArgs.framework.getMaintainer(), "Only Maintainer can perform this action");
```

Comment:
Sorry for missing this the last time. But isn't this checking if it's the maintainer twice? Since this is internal and the `init()` from `PaymentExitGame` already has a modifier ?

Reply:
yeah from discussing it with @boolafish I understood that security reviewers prefer having these checks on all layers. So I kept it. @boolafish have you changed your mind?

Reply:
Aah I see. Just more gas I suppose.

Reply:
lol if you did not introduce this at the beginning I would personally not ask you to add this. Me personally are not too much a fan of this honestly 😅 but from experience some times auditor would prefer more checks on each level even it is duplicated or not really possible to trigger.

Or probably let the security speak 😛 cc. @thec00n